### PR TITLE
Take PUBLIC_SCOPE into account during validation

### PR DESCRIPTION
**Pull request by jjmurre**
Recently, a public scope "OPENBAAR" has been added to all datasets.
The validator that validates the scopes, should take this PUBLIC_SCOPE
into account.

### CODE REVIEW

File: src/schematools/permissions/db.py [hunk 1]
```diff
@@ -9,14 +9,15 @@
...
+from schematools.permissions import PUBLIC_SCOPE
 from schematools.types import DatasetSchema
 from schematools.utils import to_snake_case
 
 # Create a module-level logger, so calling code can
 # configure the logger, if needed.
 logger = logging.getLogger(__name__)
 
-PUBLIC_SCOPE = "OPENBAAR"
+# PUBLIC_SCOPE = "OPENBAAR"
```
Comment (lbam) [inline src/schematools/permissions/db.py:20]:
```suggestion
```

Comment (jjmurre) [reply src/schematools/permissions/db.py]: Zag-em ook net :-)

File: src/schematools/permissions/db.py [hunk 2]
```diff
@@ -9,15 +9,14 @@
 from sqlalchemy.engine import Engine
 from sqlalchemy.orm import Session, sessionmaker
 
+from schematools.permissions import PUBLIC_SCOPE
```
Comment (lbam) [inline src/schematools/permissions/db.py:12]: Doesn't this lead to an import cycle? It did the last time I tried removing the constant from this file.

**Review by lbam — APPROVED** [review]

**Comment by lbam** [comment]
LGTM modulo the potential import cycle. If you can run this locally, feel free to merge.

File: src/schematools/permissions/__init__.py [hunk 3]
```diff
@@ -1,3 +1,5 @@
+"""Base package for permissions."""
```
Comment (lbam) [inline src/schematools/permissions/__init__.py:1]: Side remark, but this doc isn't very informative. If this is to satisfy a checker, maybe we should get set it to a laxer mode.

Comment (jjmurre) [reply src/schematools/permissions/__init__.py:1]: Agree.

File: src/schematools/permissions/db.py (hunk 2 above)
Comment (jjmurre) [reply src/schematools/permissions/db.py:12]: When running the tests, this module is touched, but I don't get an import cycle. Is there a way to trigger it?

File: src/schematools/permissions/__init__.py (hunk 3 above)
Comment (jjmurre) [reply src/schematools/permissions/__init__.py]: I laxed this flake8 check in setup.cfg.

File: src/schematools/permissions/db.py (hunk 2 above)
Comment (lbam) [reply src/schematools/permissions/db.py:12]: Should be ok then.

**Review by lbam — APPROVED** [review]

**Pull request merged**